### PR TITLE
Disallow nil values in git changelog action :between array

### PR DESCRIPTION
**Pull request by mfurtak**

### CODE REVIEW

File: spec/actions_specs/changelog_from_git_commits_spec.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@
         end.to raise_error(":between must be of type array".red)
       end
 
-      it "Does not accept an array of size 1" do
+      it "Does not accept a :between array of size 1" do
```
Comment (KrauseFx) [inline spec/actions_specs/changelog_from_git_commits_spec.rb]: Ups, did you intend to change this?

**Comment by KrauseFx** [comment]
Looks great, I added one comment before it's ready :rocket:

File: spec/actions_specs/changelog_from_git_commits_spec.rb (hunk 1 above)
Comment (mfurtak) [inline spec/actions_specs/changelog_from_git_commits_spec.rb]: Yep. When I added the new test I felt like those messages could use more clarification about where this "array" we were testing was coming from.

**Comment by KrauseFx** [comment]
:+1:

**Pull request merged**